deprecated/utils/templates/template.py

```diff
@@ -32,30 +33,34 @@
 
 import rospy
 import roslib
-import numpy,math,tf,threading
+import numpy
+import math
+import tf
+import threading
 from kill_handling.listener import KillListener
 from kill_handling.broadcaster import KillBroadcaster
 
 rospy.init_node('NODE_NAME')
 
+
 class CLASS_NAME(object):
-	# Base class for whatever you are writing
-	def __init__(self):
-		# Initilization code
-		self.killed = False
-		self.kill_listener = KillListener(self.set_kill, self.clear_kill)
+    # Base class for whatever you are writing
+
+    def __init__(self):
+        # Initilization code
+        self.killed = False
+        self.kill_listener = KillListener(self.set_kill, self.clear_kill)
 
     def set_kill(self):
         self.killed = True
 
     def clear_kill(self):
         self.killed = False
 
-	def state_status(self):
-		# State info to go here
+    def state_status(self):
+        # State info to go here
 
-if __name__ == "__main__":
-	#
-	CLASS_INSTANCE = CLASS_NAME()
-    rospy.Timer(rospy.Duration(1), CLASS_INSTANCE.state_status)
-    rospy.spin()
+        if __name__ == "__main__":
```

Comment:
This is not part of the class and doesn't need to be indented. (this file is deprecate anyways so w/e)